tests/multitest/metrics/conftest.py

```diff
@@ -41,6 +50,37 @@ class TestMetric:
     # additional_check: Optional[Callable[[Report], None]] = None
     # """Additional callable to call on report"""
 
+    if SUGGEST_FINGERPRINT:
+
+        def __post_init__(self):
+            import inspect
+
+            fingerprint = self.metric.get_fingerprint()
+            if self.fingerprint is None or self.fingerprint != fingerprint:
+                stack = inspect.stack()
+                init_call = stack[2]
```

Comment:
Can you add comment here why we need a inspect.stack() and it's not enough just compare fingerprint